tests/components/top_bar.test.ts

```diff
@@ -771,3 +771,71 @@ describe("Topbar - View", () => {
     app.destroy();
   });
 });
+
+describe("Topbar - menu item resizing with viewport", () => {
+  test("font size dropdown in top bar is resized with screen size change", async () => {
+    const model = new Model();
+    const { app } = await mountParent(model);
+    triggerMouseEvent('.o-tool[title="Font Size"]', "click");
+    await nextTick();
+    setTimeout(() => {
+      let height = getElComputedStyle(".o-dropdown-content.o-text-options", "maxHeight");
+      expect(parseInt(height)).toBe(
+        model.getters.getVisibleRect(model.getters.getActiveMainViewport()).height
+      );
+    }, 100);
```

Comment:
nonononono don't do that. The timeout will never be run, because it runs _after_ the test ends. Try to do a console.log in the timeOut to see it. Your test pass, but only because the `expect` is never run. You could throw errors and call the god of bugs in the timeout, and the test would still work.

The issue you were having is that you were missing the props `dropdownMaxHeight` in the mock topbar, at the top of this file.

Reply:
Update the test, please let me know if you need any changes